submissions/crealime/html-css-popup/index.html

```diff
@@ -0,0 +1,150 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <title>HTML & CSS Practice | Kottans</title>
+  <meta name="description" content="Popup menu">
+  <meta name="theme-color" content="#949494">
+  <link rel="icon" href="img/favicon.png" sizes="32x32">
+  <link href="css/main.css" rel="stylesheet">
+</head>
+<body>
+<div class="wrapper">
+  <header class="header">
+    <div class="top-menu">
+      <nav class="top-menu__nav">
+        <ul class="top-menu__ul">
+          <li class="top-menu__li">
+            <a class="top-menu__link" href="#" title="Mail">Mail</a>
+          </li>
+          <li class="top-menu__li">
+            <a class="top-menu__link" href="#" title="Images">Images</a>
+          </li>
+          <li class="top-menu__li">
+            <input class="top-menu__input" id="prod" type="checkbox" title="Products">
+            <label class="top-menu__label" for="prod">
+              <span class="top-menu__icon-box">
+                <img class="top-menu__icon" src="img/popup-button.png" alt="Products">
+              </span>
+            </label>
+            <div class="products">
+              <input class="products__input" id="more" type="checkbox" title="More">
+              <ul class="products__ul">
+                <li class="products__li">
+                  <a class="products__link" href="#" title="Calendar">
+                    <img class="products__icon" src="img/calendar.png" alt="Calendar">
```

Comment:
This is not a correct use of the `alt` attribute, please check
https://www.w3.org/QA/Tips/altAttribute

Reply:
I'm sorry. I read it, but I still don't understand what the error is. Can you explain?

Reply:
I think I got it )